gatsby/content/projects/2015-11-29-ruma.mdx

```diff
@@ -1,19 +0,0 @@
----
-layout: project
-title: Ruma
```

Comment:
For reference on Ruma being dead: https://github.com/ruma/homeserver has been archived and links to https://www.ruma.io/news/ruma-is-dead-long-live-ruma-2020-04-10/


https://matrix.org/docs/projects/try-matrix-now has a lot of other "Not actively maintained" projects but we just have that box unchecked by default.

Seems like it would be ok to remove though 🤔

Reply:
I don't know the status of the other ones, not actively maintained could also mean that it might come back at some point, which is definitely not the case for the Ruma HS (there's too much confusion with the overall project, which is also why I posted this PR).

Reply:
Imo it is better to just set it as not actively maintained rather than deleting it even if you plan to never do anything with it. It’ll be hidden by default but is still findable if someone is looking for Rust code that does Matrix things that they can use for inspiration or just as an archive of what previously Matrix projects have existed.

Reply:
I'd really prefer it to be deleted, given the somewhat common confusion with the overall project of the same name.

Reply:
For reference, @jplatte is a maintainer of Ruma. I'm inclined to go with your wishes although leaving it in "Not actively maintained" seems perfectly fine as well. I'll leave this for another day in case you want to respond, then merge.

The change to `gatsby/src/pages/faq.js` is good in any case.